### ISSUE TITLE
Format for print

### ISSUE THREAD

**Issue by rhiaro** [rhiaro/thesis#56]
* [ ] Page numbers
* [x] Tables not huge and overflowing
* [ ] break sections (tables, lists, etc) in sensible places
* [x] increase font size for print to 12pt (changed to using 10pt for default)
* [x] double line space
* [x] increase font size for level 3 section headings
* [ ] ...

**Comment by rhiaro** [comment]
* [x] Table #swwg-user-stories in chapter 5 is horrendous. Col 1 too wide, font too big, internal lists need to break across pages

**Comment by rhiaro** [comment]
* [x] Some diagrams are too enormous when they are 100% width.

**Comment by csarven** [comment]
Created issue https://github.com/linkeddata/dokieli/issues/191

**Comment by csarven** [comment]
One note on the page numbers. Most browsers can display page numbers in the header/footer area natively (see print settings). That's not entirely desirable unless it is only the page numbers. Firefox allows this e.g., print page # at the bottom right corner.

Unfortunately, this starts the counter at 1 on page 1. The first page is probably the cover of some sort.. and not all pages may need to display the number. Depends on your taste or requirements.

There is also the part dealing with different documents but as a collective I suppose. You have each chapter on its own. How do you plan to print them keeping the page numbers in mind. I think one strategy would be to have a single stand-alone document just for the purpose of the print version of your thesis that you plan to submit to your institution.

Bottom line: with CSS, we can add the page numbers but I'm not sure (at the moment) if it can do anything above and beyond what the browser's print feature can do. The main challenge right now that I can think of is to resolve show/hide the page numbers on certain pages. If we treat the cover page or some other pages (until you get to the actual content bits) separately, then this might work out okay. So, basically, you'll have to print (whether actual paper or PDF) them separately. That way, you can start the page numbering from where you want it to start. You'll have to separately join all of the separate documents (i.e., cover, acknowledgements... each chapter) into one one document.

Browser support for print CSS is indefinitely poop.

**Comment by rhiaro** [comment]
* [x] is there a way to automatically make hyperlinks into nice looking footnotes?

**Comment by csarven** [comment]
Do you mean to show simple hyperlinks as footnotes.. (at the bottom of respective page when printed) instead of in references / end notes?

**Comment by rhiaro** [comment]
yes

**Comment by rhiaro** [comment]
* [x] Make h1 and h2 styles look how they are in my original version
* [ ] Some headers are still orphaned at the bottom of pages
* [x] hide `.notes`
* [x] hyperlinks need to show up *somewhere* whether in brackets inline or as footnotes

**Comment by csarven** [comment]
* [x] Normalise footnote id display

Since the footnote bit is in `aside`, it should accommodate any HTML. That is, not all information would need to start off with an id. The aside could merely be commentary.

For the existing id like info, perhaps normalise the ones that are in use e.g., using `foo.`, `[foo]`, `<strong>[foo]</strong>`, `<strong>[foo]:</strong>`, or maybe even consider `<sup>foo</sup>` again as it would be similar to the usage in the referred text.

Decided to use `<sup>foo</sup>` for now as default. Updated all docs.
